mono/eglib/gunicode-win32-uwp.c

```diff
@@ -11,24 +11,24 @@
 #define CODESET 1
 #include <windows.h>
 
-extern const char *my_charset;
+extern const char *eg_my_charset;
 static gboolean is_utf8;
```

Comment:
I think a lot of these symbols should be made static instead of renaming them.

Reply:
I made symbols `static` where doing so was possible without further code change (such as exposing a getter function). This one in particular is, surprisingly, used across multiple files in eglib.